File: AR-VR/priyanshurat_PriyanshuRathaur_2024csit1047_2/GettingStarted/GradingSystem.cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>

using namespace std;

int main()
{
   int a;
   cin>>a;
   
   if (a>90)
   {cout<<"excellent";}
  else if (a<=90 && a>80)
   {cout<<"good";}
   else if (a<=80 && a>70)
   {cout<<"fair";}
   
   else if (a<=70 && a>60)
   {cout<<"meets expectations";}
   
    else
   {cout<<"below par";}

    return 0;
}

File: AR-VR/Vidushi7555_VidushiPandey_2024ec1126_2/GettingStarted/GradingSystem.java
/******************************************************************************

                            Online Java Compiler.
                Code, Compile, Run and Debug java program online.
Write your code in this editor and press "Run" button to execute it.

*******************************************************************************/

import java.util.*;
  
  public class Main
  {
  
  public static void main(String[] args)
  {
      // input - don't change this code
      Scanner scn = new Scanner(System.in);
      int marks = scn.nextInt();
      if(marks>90)
      {
          System.out.println("excellent");
      }
      else if((marks>80)&&(marks<=90))
      {
          System.out.println("good");
      }
      else if((marks>70)&&(marks<=80))
      {
          System.out.println("fair");
      }
      else if((marks>60)&&(marks<=70))
      {
          System.out.println("meets expectations");
      }
      else
      {
          System.out.println("below par");
      }
      
      
      
  
   }
  }
